src/modules/GameConstants.ts

```diff
@@ -360,6 +360,16 @@ export enum Currency {
     contestToken,
 }
 
+export const LuxuryBallCurrencyRate: Record<Currency, number> = {
+    [Currency.money]: 200000,
+    [Currency.questPoint]: 60,
+    [Currency.dungeonToken]: 15000,
+    [Currency.diamond]: 1,
+    [Currency.farmPoint]: 60,
+    [Currency.battlePoint]: 10,
+    [Currency.contestToken]: 60,
```

Comment:
The previous balance suggestion didn't take into account that achievement % is only applying to money and dt, could you bump up the numbers on the rest to bring it more in line with what was intended?
```suggestion
export const LuxuryBallCurrencyRate: Record<Currency, number> = {
    [Currency.money]: 200000,
    [Currency.questPoint]: 600,
    [Currency.dungeonToken]: 15000,
    [Currency.diamond]: 10,
    [Currency.farmPoint]: 600,
    [Currency.battlePoint]: 100,
    [Currency.contestToken]: 600,
```

Reply:
Done. Tell me if they feel fine like that.